app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  validates :skill_level, inclusion: { in: %w(beginner intermediate expert) }
+
+  has_many :subscriptions
```

Comment:
change lines 4 and 5 to has_many :posts, dependent: :destroy
  has_many :subscriptions, dependent: :destroy
  you must include the depend destroy or you will get a foreign key error when trying to delete users